INSTALL.md

```diff
@@ -43,7 +43,7 @@ That's it! *Honestly*!
 
 #### Windows
 On Windows you can download an executable for Bitmessage
-[here](https://bitmessage.org/download/windows/Bitmessage.exe).
+[here](https://github.com/Bitmessage/PyBitmessage/releases/download/0.6.3.2/Bitmessage_x86_0.6.3.2.exe).
```

Comment:
Better would be to point to releases without a version.

Reply:
Although since the parent repo doesn't seem to exist anymore, maybe this can be done separately.